ckanext/fcscopendata/templates/package/group_list.html

```diff
@@ -7,6 +7,7 @@ <h2 class="hide-heading">{{ _('Groups') }}</h2>
   {% if group_dropdown %}
     <form class="add-to-group" method="post">
       <select id="field-add_group" name="group_added" data-module="autocomplete">
+        {% set group_dropdown = h.get_dataset_group_list() %}
         {% for option in group_dropdown %}
```

Comment:
@MuhammadIsmailShahzad it should remove the already selected group item from the dropdown option.